README.md

```diff
@@ -1,38 +1,53 @@
-What is Notepad++ ?
-===================
+# What is Notepad++ ?
 
 [![Join the chat at https://gitter.im/notepad-plus-plus/notepad-plus-plus](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/notepad-plus-plus/notepad-plus-plus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 Notepad++ is a free (free as in both "free speech" and "free beer") source code editor and Notepad replacement that supports several programming languages and natural languages. Running in the MS Windows environment, its use is governed by GPL License.
 
 
-To build Notepad++ package from source code:
-============================================
+## To build Notepad++ from source:

```

Comment:
I don't have any preferences for that. It was just for `h1` and `h2` tags. And ~80 columns in the same time if possible.